### PR TITLE
Add insights tab

### PR DESCRIPTION
**Pull request by MehdiElrafi**

### CODE REVIEW

**Comment by MehdiElrafi** [comment]
image

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -48,14 +47,17 @@ def destroy
...
   end
 
+  def insights
+  end
+
   private
 
   def set_breadcrumbs
     add_breadcrumb(t("views.layouts.main.hiring"), jobs_path)
+    add_breadcrumb(Job.find(@job.id).title, job_path(@job.id))
```
Comment (itsAndL) [inline app/controllers/jobs_controller.rb:60]: `@job.title` instead of `Job.find(@job.id).title`

File: app/views/shared/_job_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<div class="<% unless user_signed_in? %>mt-5 <% end %> sm:flex-auto">
+  <h1><%= t("views.jobs.more_details") %> <%= job.title %></h1>
+</div>
+
+<% if user_signed_in? %>
+  <div class="sm:flex sm:flex-row gap-5 mt-5">
+    <div class="sm:flex w-60 p-6 bg-white border border-gray-200 rounded-lg shadow hover:bg-gray-100">
+      <div class="mb-2 mr-4 text-2xl font-bold tracking-tight text-gray-900">
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-12 h-12">
```
Comment (itsAndL) [inline app/views/shared/_job_header.html.erb:9]: put the svgs in `images/icons` folder

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (mohamedrez) [reply app/controllers/jobs_controller.rb:60]: yes it is an extra query

**Review by itsAndL — APPROVED** [review]

**Pull request merged**